packages/bot/src/transformers/sku.ts

```diff
@@ -1,4 +1,4 @@
-import type { DiscordSku, DiscordSkuFlag, DiscordSkuType } from '@discordeno/types'
+import type { DiscordSku, DiscordSkuType, SkuFlag } from '@discordeno/types'
```

Comment:
Inconsistency here

Reply:
The `Discord`? If so, there is a question but you ignored it from what i can tell. For now i removed it bc other enums are this way (`UserFlags` or `ChannelFlags`)